packages/plugin-core/src/commands/CopyToClipboardCommand.ts

```diff
@@ -0,0 +1,41 @@
+import { window } from "vscode";
+import { DENDRON_COMMANDS } from "../constants";
+import { clipboard } from "../utils";
+import { BasicCommand } from "./base";
+
+type CopyToClipboardCommandOpts = {
+  text: string;
+  message?: string;
+};
+
+/**
+ * This command is not accessible through the VSCode UI,
+ * and only intended to be used as a proxy for copying arbitrary
+ * text from the webview.
+ *
+ * e.g.)
+ *
+ * // you can use this in a markdown link to invoke commands
+ * const commandUri = `command:dendron.copyToClipboard?${encodeURIComponent({
+ *   text: "some text",
+ *   message: "copied!"
+ * })}`
+ *
+ * ...
+ *
+ * content = `[click this](${commandUri})`
+ *
+ */
+export class CopyToClipboardCommand extends BasicCommand<
```

Comment:
can you add this to [[Cook|dendron://dendron.docs/pkg.plugin-core.dev.cook]]?

Reply:
If you check the [docs PR](https://github.com/dendronhq/dendron-docs/pull/22), I have them in there. I also created an async.

Reply:
I can add a bit more about `CopyToClipboardCommand` itself if you think the docs as-is is a bit lacking.